src/scope.h

```diff
@@ -113,7 +113,9 @@ struct Scope
     int explicitProtection;     // set if in an explicit protection attribute
 
     StorageClass stc;           // storage class
-    char *depmsg;               // customized deprecation message
+
+    // customized deprecation message
+    DeprecatedDeclaration *depdecl;
```

Comment:
Please use one line, not 3. We're not introducing a new style for how fields are commented.